src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular/nswag.json

```diff
@@ -60,7 +60,7 @@
         "SchemaService",
         "ShareService"
       ],
-      "extensionCode": "src/lib/services.extensions",
+      "extensionCode": "src/lib/services.extensions.ts",
```

Comment:
- Add ListItemsService.get, ContentService.create, ContentService.updateMetadata and ContentService.updatePermissions, ShareService.get to NSwag.json and services.extensions.ts
- Remove SchemaService

Reply:
Done